docs/overrides.css

```diff
@@ -0,0 +1,3 @@
+/*.toctree-l1 li { display: none }*/
```

Comment:
Honestly, let's just merge this in and iterate. It's too small to hold up the PR.